Ladybird/Settings.cpp

```diff
@@ -23,4 +23,14 @@ void Settings::set_homepage(QString const& homepage)
     m_qsettings->setValue("homepage", homepage);
 }
 
+bool Settings::go_to_homepage_on_init()
+{
+    return m_qsettings->value("go_to_homepage_on_init", true).toInt() == 1;
+}
+
+void Settings::set_go_to_homepage_on_init(bool const& go_to_homepage_on_init)
+{
+    m_qsettings->setValue("go_to_homepage_on_init", int(go_to_homepage_on_init));
+}
```

Comment:
I'm not sure this makes sense. Why would you specify a homepage and then instruct the browser to not load it?